ext/popup.js

```diff
@@ -44,22 +50,44 @@ Workspace.all = function() {
 
 Workspace.open = function(workspace) {
   chrome.storage.local.get(workspace.name, function(bucket) {
-    chrome.windows.create({ url: bucket[workspace.name] });
+    chrome.windows.create(function(newWindow) {
+      let isFirstTabLoad = false;
+
+      bucket[workspace.name].forEach((tabInfo) => {
```

Comment:
This is a blocker for this PR, we only have a few users but I dont want to update and destroy all their precious workspaces.